snippets/ibu-ImageBasedGroupUpgrade.adoc

```diff
@@ -0,0 +1,41 @@
+[source,yaml]
+----
+apiVersion: lcm.openshift.io/v1alpha1
+kind: ImageBasedGroupUpgrade
+metadata:
+  name: example-group-upgrade
+  namespace: default
+spec:
+  clusterLabelSelectors: <1>
+  - matchExpressions:
+    - key: name
+      operator: In
+      values:
+      - spoke1
+      - spoke4
+      - spoke6
+  ibuSpec:
+    seedImageRef: <2>
+      image: quay.io/seed/image:4.16.0-rc.1
+      version: 4.16.0-rc.1
+      pullSecretRef:
+        name: "<seed_pull_secret>"
+  extraManifests: <3>
+    - name: example-extra-manifests
+      namespace: openshift-lifecycle-agent
+  oadpContent: <4>
+    - name: oadp-cm
+      namespace: openshift-adp
```

Comment:
More indentation. Did we fix this in 4.17?

Reply:
I just checked, doesn't seem to be correct in 4.17. I'll open a bug for that version and track it there but I'll fix it for 4.16 in this PR.